src/resourcesBuilder.js

```diff
@@ -0,0 +1,51 @@
+import {Delete, Resource} from 'admin-on-rest';
```

Comment:
We already have two factories: `inputFactory` and `fieldFactory`. I suggest you to rename this file `resourceFactory`.

Reply:
I'll refer to your both comments here, IMO factory should return only one item and obviously there it would return array of `Resource` components. Maybe then we should rename it indeed to `resourceFactory` but export only what's currently in code `const makeResource`? WDYT?